<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Record</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background:rgb(210,232,237);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Code :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtStudCode">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Last Name :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  font-family: &quot;Roboto&quot;, sans-serif;
  text-transform: uppercase;
  outline: 0;
  background:rgb(255,192,0);
  width: 100%;
  border: 0;
  padding: 0px;
  color: #000;
  font-size: 12px;
  border-radius:10px;
}
QPushButton:hover{
	color:white;
	background-color:rgb(239,93,93);
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  font-family: &quot;Roboto&quot;, sans-serif;
  text-transform: uppercase;
  outline: 0;
  background:rgb(255,192,0);
  width: 100%;
  border: 0;
  padding: 0px;
  color: #000;
  font-size: 12px;
  border-radius:10px;
}
QPushButton:hover{
	color:white;
	background-color:rgb(239,93,93);
}</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  font-family: &quot;Roboto&quot;, sans-serif;
  text-transform: uppercase;
  outline: 0;
  background:rgb(255,192,0);
  width: 100%;
  border: 0;
  padding: 0px;
  color: #000;
  font-size: 12px;
  border-radius:10px;
}
QPushButton:hover{
	color:white;
	background-color:rgb(239,93,93);
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  font-family: &quot;Roboto&quot;, sans-serif;
  text-transform: uppercase;
  outline: 0;
  background:rgb(255,192,0);
  width: 100%;
  border: 0;
  padding: 0px;
  color: #000;
  font-size: 12px;
  border-radius:10px;
}
QPushButton:hover{
	color:white;
	background-color:rgb(239,93,93);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>First Name :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Middle Name :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableStudents">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>511</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Student Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </column>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtStudCode</tabstop>
  <tabstop>txtLName</tabstop>
  <tabstop>txtFName</tabstop>
  <tabstop>txtMName</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>tableStudents</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
